lib/galaxy/datatypes/media.py

```diff
@@ -287,3 +356,66 @@ def set_meta(self, dataset: DatasetProtocol, overwrite: bool = True, **kwd) -> N
                 dataset.metadata.nchannels = fd.getnchannels()
         except wave.Error:
             pass
+
+
+class Ogg(Audio):
+    file_ext = "ogg"
+
+    def sniff(self, filename: str) -> bool:
+        return _get_file_format_from_magic_number(filename, "ogg")
+
+
+class Webm(Video):
+    file_ext = "webm"
+
+    def sniff(self, filename: str) -> bool:
+        if which("ffprobe"):
+            metadata, streams = ffprobe(filename)
+            vp_check = any(
+                stream["codec_name"] in ["vp8", "vp9"] for stream in streams if stream["codec_type"] == "video"
```

Comment:
webm also supports av1.

Reply:
So, should I check `av1`, `vp8`, and `vp9` for `webm` or is this approach entirely incorrect?

Reply:
Based on https://github.com/galaxyproject/galaxy/pull/18054#discussion_r1579499848, I don't know. 

Perhaps yes, check for those formats and if they're present it's a "webm". I think we could add a `set_peek` method which says "WebM subset of MKV" (or a similar description) to communicate that to the user that it's still actually a matroshka container.